Update greeting module tests

```
======================================================================
FAIL: test_greeting_remove_m_hint (phenny.modules.test.test_greeting.TestGreeting)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/apertium/phenny/modules/test/test_greeting.py", line 55, in test_greeting_remove_m_hint
    self.phenny.msg.assert_called_once_with(self.input.nick, self.input.nick + ": " + hint)
  File "/home/travis/virtualenv/python3.8.0/lib/python3.8/site-packages/mock/mock.py", line 957, in assert_called_once_with
    return self.assert_called_with(*args, **kwargs)
  File "/home/travis/virtualenv/python3.8.0/lib/python3.8/site-packages/mock/mock.py", line 944, in assert_called_with
    six.raise_from(AssertionError(_error_message(cause)), cause)
  File "<string>", line 3, in raise_from
AssertionError: expected call not found.
Expected: msg('Test[m]worth', 'Test[m]worth: Consider removing [m] from your IRC nick! See http://wiki.apertium.org/wiki/IRC/Matrix#Remove_.5Bm.5D_from_your_IRC_nick for details.')
Actual: msg('Test[m]worth', 'Test[m]worth: Please consider removing [m] from your IRC nick. See http://wiki.apertium.org/wiki/IRC/Matrix#Remove_.5Bm.5D_from_your_IRC_nick for details. Reply .dismiss to prevent this message from appearing again.')
```

--- a/modules/test/test_greeting.py
+++ b/modules/test/test_greeting.py
@@ -51,5 +51,5 @@ class TestGreeting(unittest.TestCase):
 
         greeting.greeting(self.phenny, self.input)
 
-        hint = "Consider removing [m] from your IRC nick! See http://wiki.apertium.org/wiki/IRC/Matrix#Remove_.5Bm.5D_from_your_IRC_nick for details."
+        hint = "Please consider removing [m] from your IRC nick. See http://wiki.apertium.org/wiki/IRC/Matrix#Remove_.5Bm.5D_from_your_IRC_nick for details. Reply .dismiss to prevent this message from appearing again."
         self.phenny.msg.assert_called_once_with(self.input.nick, self.input.nick + ": " + hint)

